bundles/net.tourbook.statistics/src/net/tourbook/statistics/messages.properties

```diff
@@ -42,23 +42,27 @@ numbers_info_time_down        = Duration\u0009<%sh * %d
 numbers_info_time_total       = Total\u0009%sh
 numbers_info_time_up          = Duration\u0009>%sh * %d
 
-tourtime_info_altitude            = Elevation\u0009\u0009%d %s
-tourtime_info_avg_pace            = \u00F8 Pace\u0009\u0009%d:%02d %s
-tourtime_info_avg_speed           = \u00F8 Speed\u0009\u0009%1.1f %s
-tourtime_info_break_time          = Paused Time\u0009%d:%02d h
-tourtime_info_break_time_tour     = Paused Time\u0009%d:%02d:%02d h
-tourtime_info_date_month          = %s - %s %d
-tourtime_info_date_week           = %s - %s
-tourtime_info_date_year           = %s
-tourtime_info_description         = Description
-tourtime_info_description_text    = %s
-tourtime_info_distance            = Distance\u0009\u0009%d %s
-tourtime_info_distance_tour       = Distance\u0009\u0009%1.1f %s
-tourtime_info_driving_time        = Moving Time\u0009%d:%02d h
-tourtime_info_driving_time_tour   = Moving Time\u0009%d:%02d:%02d h
-tourtime_info_recording_time      = Recording Time\u0009%d:%02d h
-tourtime_info_recording_time_tour = Recording Time\u0009%d:%02d:%02d h
-tourtime_info_tags                = Tags\u0009\u0009%s
-tourtime_info_time                = Time\u0009\u0009%d:%02d - %d:%02d
-tourtime_info_tour_type           = Type\u0009\u0009%s
-tourtime_info_week                = %s - CW %d / %d 
+tourtime_info_altitude           = Elevation\u0009\u0009%d %s
+tourtime_info_avg_pace           = \u00F8 Pace\u0009\u0009%d:%02d %s
+tourtime_info_avg_speed          = \u00F8 Speed\u0009\u0009%1.1f %s
+tourtime_info_break_time         = Paused Time\u0009%d:%02d h
+tourtime_info_break_time_tour    = Paused Time\u0009%d:%02d:%02d h
+tourtime_info_date_month         = %s - %s %d
+tourtime_info_date_week          = %s - %s
+tourtime_info_date_year          = %s
+tourtime_info_description        = Description
+tourtime_info_description_text   = %s
+tourtime_info_distance           = Distance\u0009\u0009%d %s
+tourtime_info_distance_tour      = Distance\u0009\u0009%1.1f %s
+tourtime_info_elapsed_time       = Elapsed Time\u0009%d:%02d h
+tourtime_info_elapsed_time_tour  = Elapsed Time\u0009%d:%02d:%02d h
+tourtime_info_moving_time        = Moving Time\u0009%d:%02d h
+tourtime_info_moving_time_tour   = Moving Time\u0009%d:%02d:%02d h
+tourtime_info_paused_time        = Paused Time\u0009%d:%02d h

```

Comment:
Break time label is "Paused Time"

`tourtime_info_break_time = Paused Time`
`tourtime_info_paused_time = Paused Time`

Reply:
Fixed